### DISCUSSION TITLE
CTAs position on cards

### DISCUSSION THREAD

**Discussion by MontseTelefonica** [Telefonica/mistica-design#802] [Proposals]
As cards in carrousels take the height of the largest one to maintain a consistent look, we believe that CTAs should be placed on the same position on the cards, fixed to the bottom.

<img width="778" alt="image" src="https://user-images.githubusercontent.com/95479574/192741643-7e08a325-3a2d-4393-a196-47587529cb1b.png">

**Comment by aweell** [comment]
Hey! @MontseTelefonica,

This is a beautiful example of how things working in code are a headache when being reproduced using design tools like Figma.

Expect your proposal [to work as intended when grouping cards](https://mistica-j04vr6okj-tuentisre.vercel.app/playroom#?code=N4Igxg9gJgpiBcIA8AFATjAbgSxgdwBUIIAbAZwD4kB6dLXQ48igHQDskAlGMgBwjZlsmGABkAhgE8IAVwAurNgAIlSKMPYqVZOZJIwAvMGCatK9XxJT4SliADmabFDsAaU2cfOCMALa8rORgAYVIZX0EbOwBGADM0JTiEpLcPLXtxXiiQaIxfVOUtAF8i00UzJAAJbHsACxIa2qCoYPE0F0KzOWw5fQM7Ah79OzTzHjAnXm6BfpAAZQhfGCVYMgnsKewBJVqYDFg8AHEAMQApAFUAIQ37WUPKy4B5AE3avAhMGVqyWVqANQAS3NxLVMBAyDIQZh7CNOlpsL5xPYYOc0CRZk05LwyPBqNRsAA6BH2GRoAmQXzUABWMAAInMABqXc4Eqm8GEgUZKATBEjgwzAAAUAEolAYKEpgCUuQAjeRyGYmOEVS7ygToBFtSQ7DCxWYAYjsSjIiJIJHKZktqrkCrYGsRaEkXJUNGttvtWotZlKcOoXtU1TqDTqzVa7S53V6hgGQxgsMtK3Gk2mbFmCyWibWya2yl2GHjls1yNR6LsmOxuPxRN8JLJFOpdMZzNZ7NhXJ5fLIApFYolUp9CblNsVztUbvVTgd2tquoNRpN4jN-oT47tk61o9darXmsdy6UA60ftMNHUmEUNG4fAEQhEEmk8kUIFcIDkuyWZAQAG0QJcrDJnxAABZD5sB0NpAMeAAmQCfH0WIBGwMBxEAv5hAgEAAF0XzwZw30-eAv2iABmAA2AAOTCiiAA):

![Screenshot 2022-10-18 at 14 03 29](https://user-images.githubusercontent.com/44420072/196424367-08a75822-ad3c-4fc3-909b-c02b81443e9e.png)

![Screenshot 2022-10-18 at 14 09 13](https://user-images.githubusercontent.com/44420072/196425435-ab1f8cb4-acdb-4f04-81a1-9781f00ee30a.png)

I'm trying to figure out what's the best override approach to provide a better representation of the behavior when grouping cards.

**Comment by MontseTelefonica** [comment]
These screenshots look good to me. What is the problem :) ?

**Comment by yceballost** [comment]
We agree with this evolution, thanks Montse!

**Reply by yceballost** [reply]
Was a bug in web! It is already fixed and will be in Novum in the next Mística update!
